getting-started/blazor/rendering-different-charts-using-blazor.md

```diff
@@ -0,0 +1,10 @@
+rendering-different-charts-using-blazor.md
+---
+title: Rendering different Charts | FusionCharts
+description: This article outlines some of the popular charts and the way to render them with their respective data formats.
+heading: Rendering Different Charts
+---
+
+In the [Getting Started](/getting-started/flutter/your-first-chart-using-flutter) section, we discussed building a single series chart using FusionCharts Suite XT. In this section, you can see how to create different charts and maps using FusionCharts and Blazor.
```

Comment:
flutter reference

Reply:
Page was updated with examples and this correction was made too